dlme_airflow/tasks/transform_validation.py

```diff
@@ -6,6 +6,10 @@
 from airflow.utils.task_group import TaskGroup
 
 
+def eval_record_count_formula(harvested_record_count, record_count_formula):
+    return eval(f"{harvested_record_count}{record_count_formula}")
```

Comment:
I think we may want to wrap this in a `try` block to avoid bad configuration causing failures...

Would you consider something like:

```
try:
    return eval....
except ZeroDivisionError:
    return 0
```

That catches a divide by zero error, there may be others so we'd want a more general `except` but this may be sufficient for now.